### PR TITLE
Minor code tweaks/optimizations (PHP)

### PR DESCRIPTION
**Pull request by aristath**
## What?
Just some minor code optimizations. Details in comments for each change.

### CODE REVIEW

File: lib/block-supports/border.php [hunk 1]
```diff
@@ -12,22 +12,18 @@
  * @param WP_Block_Type $block_type Block Type.
  */
 function gutenberg_register_border_support( $block_type ) {
-	// Determine if any border related features are supported.
-	$has_border_support       = block_has_support( $block_type, array( '__experimentalBorder' ) );
-	$has_border_color_support = gutenberg_has_border_feature_support( $block_type, 'color' );
```
Comment (aristath) [inline lib/block-supports/border.php:17]: These are only used once, so we don't need to assign them to a var.

File: lib/block-supports/border.php [hunk 2]
```diff
@@ -48,9 +44,7 @@ function gutenberg_apply_border_support( $block_type, $block_attributes ) {
 		return array();
 	}
 
-	$sides               = array( 'top', 'right', 'bottom', 'left' );
```
Comment (aristath) [inline lib/block-supports/border.php:51]: `$sides` is only used once, no need to assign it to a var

File: lib/block-supports/colors.php [hunk 3]
```diff
@@ -11,10 +11,7 @@
  * @param WP_Block_Type $block_type Block Type.
  */
 function gutenberg_register_colors_support( $block_type ) {
-	$color_support = false;
-	if ( property_exists( $block_type, 'supports' ) ) {
-		$color_support = _wp_array_get( $block_type->supports, array( 'color' ), false );
-	}
+	$color_support                 = property_exists( $block_type, 'supports' ) ? _wp_array_get( $block_type->supports, array( 'color' ), false ) : false;
```
Comment (aristath) [inline lib/block-supports/colors.php:14]: Short and sweet

**Review by ntsekouras — APPROVED** [review]
Looks good, thanks Ari!

**Pull request merged**